app/views/status_messages/new.html.haml

```diff
@@ -7,7 +7,8 @@
 :javascript
   $(function() {
     app.publisher = new app.views.Publisher({
-      standalone: true
+      standalone: true,
+      prefillMention: #{@person.to_json}
```

Comment:
Yes, very much so. I'd love if we could get rid of the inline JS and pass the data over gon...